notebooks/mock-notebooks/login/User Registration.ipynb

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
<p>In this case, if there's an error we may not have a support email, etc. As with the login notebook, we probably want to provide some general troubleshooting steps (check your URL, check in with the source of the URL) and leave it at that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>